content/context-resource/index.md

```diff
@@ -0,0 +1,320 @@
+[issue #17](https://github.com/openregister/registers-rfcs/issues/17)).
+
+```elm
+type Status
+  = Active { startDate : Timestamp }
+  | Retired { startDate : Timestamp
+            , endDate : Timestamp
+            , replacement : Maybe Url
+            , reason : Text
+            }
+```
+
+|Name|Description|
+|-|-|
+|Start date| Date when the register started.|
+|End date| Date when the register was retired.|
+|Replacement| A URL to a register that replaces the current one.|
+|Reason| A human readable explanation of why the register is no longer active.|
+
+
+#### Title
+
+* Type: Optional String
+
+The human readable name of the register.
+
+
+### HTTP resource
+
+***
+#### Endpoint
+
+```
+GET /context
+```
+
+#### Parameters
+
+|Name|Type|Description|
+|-|-|-|
+|`total-entries` | Optional `Integer`| The log size to compute the snapshot from. Note this only applies when the metadata can be versioned.|
+
+
+#### Response attributes
+
+|Name|Type|
+|-|-|
+|`id`| Name |
+|`copyright`| String |
+|`custodian`| Optional String |
+|`description`| Optional String |
+|`hashing-algorithm`| HashingAlgorithm |
+|`licence`| String |
+|`root-hash`| Hash |
+|`schema`| List Attribute |
+|`stats`| Stats |
+|`status`| Status |
+|`title`| Optional String |
+
+#### Hashing algorithm attributes
+
+|Name|Type|
+|-|-|
+|`digest-length`| Integer |
+|`function-type`| Integer |
+|`id`| Name |
+
+See also: [Multihash table function type
+identifiers](https://github.com/multiformats/multihash/blob/master/hashtable.csv).
+
+#### Attribute attributes
+
+|Name|Type|
+|-|-|
+|`id`| Name |
+|`datatype`| Datatype |
+|`cardinality`| Cardinality |
+|`title`| Optional String |
+|`description`| Optional String |
+
+#### Stats attributes
+
+|Name|Type|
+|-|-|
+|`total-entries`| Integer |
+|`total-items`| Integer |
+|`total-records`| Integer |
+
+#### Status attributes
+
+|Name|Type|
+|-|-|
+|`start-date`| Datetime |
+|`end-date`| Optional Datetime |
+|`replacement`| Optional Url |
+|`reason`| Optional String |
+
+***
+
+***
+**EXAMPLE:**
+
+
+```http
+GET /summary HTTP/1.1
+Accept: application/json
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "id": "multihash",
+  "title": "The Multihash register",
+  "description": "List of multihash codes.",
+  "custodian": "IPFS team",
+  "hashing-algorithm": {
+    "id": "sha2-256",
+    "function-type": 18,
+    "digest-length": 32
+  },
+  "stats": {
+    "total-entries": 0,
+    "total-items": 0,
+    "total-records": 0,
+  },
+  "copyright": "Copyright (c) 2016 Protocol Labs Inc.",
+  "licence": "MIT",
+  "root-hash": "1220e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+  "status": { "start-date": "2018-12" },
+  "schema": [
+    {"id": "name", "datatype": "name", "cardinality": "1"},
+    {"id": "function-type", "datatype": "integer", "cardinality": "1"},
+    {"id": "digest-length", "datatype": "integer", "cardinality": "1"}
+  ]
+}
+```
+
+***
+
+### Considerations
```

Comment:
Does this have any implications for what will happen to the register register? Is there a mechanism to ensure the register register stays in sync with what the registers themselves report for this metadata?

Reply:
Nothing concrete just yet, but we have to think about this soon.